massa_proxy/apps/massa_proxy/lib/massa_proxy/application.ex

```diff
@@ -1,10 +1,47 @@
 defmodule MassaProxy.Application do
   use Application
+  alias Vapor.Provider.{Env, Dotenv}
 
   @impl true
   def start(_type, _args) do
+    load_system_env()
+
     ExRay.Store.create()
     Metrics.Setup.setup()
     MassaProxy.Supervisor.start_link([])
   end
+
+  defp load_system_env() do
+    providers = [
+      %Dotenv{},
+      %Env{
+        bindings: [
+          {:proxy_port, "PROXY_PORT", default: 9000, map: &String.to_integer/1, required: false},
+          {:proxy_http_port, "PROXY_HTTP_PORT",
+           default: 9001, map: &String.to_integer/1, required: false},
+          {:user_function_host, "USER_FUNCTION_HOST", default: "0.0.0.0", required: false},
+          {:user_function_port, "USER_FUNCTION_PORT",
+           default: 8080, map: &String.to_integer/1, required: false},
+          {:user_function_uds_enable, "PROXY_UDS_MODE", default: false, required: false},
+          {:user_function_sock_addr, "PROXY_UDS_ADDRESS",
+           default: "/var/run/cloudstate.sock", required: false},
```

Comment:
Our proxy has the ability to communicate with the user role via unix domain sockets, in theory faster than going over the network. This can be seen and tested using the Python support language (not via branch master). There was a long discussion on this topic in some cloudstate forum (gitter or github) that I don't remember exactly anymore.

This is a path to the domain socket in the host

Reply:
I remember the discussion regarding the unix domain sockets. Thanks for the explanation :)